NitroxServer/ConsoleCommands/LoadBatchCommand.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NitroxModel.DataStructures.Util;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.GameLogic.Entities.Spawning;
+using NitroxModel.DataStructures.GameLogic;
+
+namespace NitroxServer.ConsoleCommands
+{
+    class LoadBatchCommand : Command
```

Comment:
Might be helpful to add a block comment when this command is useful.

Reply:
Yea I mainly used it for testing purposes otherwise it can likely be used to "pregen" the world